packages/ps2/src/views/Balance/components/MyBalancesTable/index.tsx

```diff
@@ -143,16 +144,31 @@ const MyBalancesTable = (): JSX.Element => {
           valueUSD: {
             balanceTotalUSDT: balance.balanceTotalUSDT,
           },
-          action: !!allowedDeposits[exchangeType]?.includes(coin) && (
-            <IconButton
-              icon={<AddIcon color={'neutral300'} />}
-              onClick={() =>
-                showModal(DepositModal, {
-                  selectedCoin: coin,
-                })
-              }
-              variant='secondary'
-            />
+          action: (
+            <>
+              {!!allowedDeposits[exchangeType]?.includes(coin) && (
+                <IconButton
+                  icon={<Add color={'neutral300'} />}
+                  onClick={() =>
+                    showModal(DepositModal, {
+                      selectedCoin: coin,
+                    })
+                  }
+                  variant='secondary'
+                />
+              )}
+              {parseFloat(balance.balanceTotal) > 0 && (
```

Comment:
I think that'd be enough
```suggestion
              {+balance.balanceTotal > 0 && (
```